json2csv.py

```diff
@@ -0,0 +1,66 @@
+import click
+import json
+import csv
+import logging
+import re
+
+
+logger = logging.getLogger(__name__)
+
+DATASET_KEY = 'nationalrat'
+SOURCE_ID = 'open_parliament_at'
+
+WANTED_FIELDS = ['first_name', 'last_name', 'salutation', 'title', 'political_affiliation',
+                 'state', 'occupation', 'place_of_birth', 'date_of_birth']
+
+party_extract_re = re.compile(r'^.* \((?P<party>\w+)\)$')
+
+@click.command()
+@click.argument('input', type=click.File('rb'))
+@click.argument('output', type=click.File('w'))
+def convert_to_csv(input, output):
+    scraped = json.load(input)
+
+    def sort_by_id(item):
+        return int(item["id"])
+    sorted_input = sorted(scraped, key=sort_by_id)
+
+    contacts = []
+    for mp_data in sorted_input:
+        mp = {
+            'identifier': mp_data['id'],
+            'source': SOURCE_ID,
+        }
+        emails = mp_data.pop('emails')
+        if len(emails) > 0:
+            mp['email'] = emails[0]
+        else:
+            mp['email'] = ''
+            logger.warning('Contact without email: %s (%s %s)', mp_data['id'], mp_data['first_name'], mp_data['last_name'])
+        mp.update({k: mp_data[k] for k in WANTED_FIELDS})
+        committees = sorted([c['name'] for f in mp_data['committees'].values() for c in f.values()])
+        mp['committees'] = ',' + ','.join(committees) + ','
+        mandates = [m['title'] for m in mp_data['mandates']]
+        # extract party from political_affiliation field
+        party = party_extract_re.match(mp_data['political_affiliation'])
+        if party.group('party'):
+            mp['party'] = party.group('party')
+        else:
+            mp['party'] = 'Unbekannt'
```

Comment:
Is this something that does occur regularly or should this trigger a notification as well?